app/src/organisms/Devices/HeaterShakerWizard/TestShake.tsx

```diff
@@ -239,7 +241,9 @@ export function TestShake(props: TestShakeProps): JSX.Element {
           </TertiaryButton>
         </Flex>
         <Flex flexDirection={DIRECTION_ROW} alignItems={ALIGN_FLEX_START}>
-          <Text width={'22rem'}>{t('troubleshoot_step2_description')}</Text>
+          <StyledText width={'22rem'} fontSize={TYPOGRAPHY.fontSizeH2}>
```

Comment:
```suggestion
          <StyledText width="22rem" fontSize={TYPOGRAPHY.fontSizeH2}>
```

Do you think we can remove the hardcode size from `StyledText`?

Reply:
yup! removing it now. thanks for catching it